### ISSUE TITLE
This is using way too much CPU...

### ISSUE THREAD

**Issue by GemExperience** [royshil/obs-backgroundremoval#70]
Please use AMD & NVIDIA encoder, not the CPU.

THANKS

**Comment by steepestascent** [comment]
@GemExperience I think windows gpu is set to build with cmake. You need to build with `-DWITH_CUDA=ON`. On linux this PR 
 #57 enables building for GPU.

**Comment by CraftingInC** [comment]
I'm using the pre-built binaries for 64-Bit Windows. I set it to GPU DirectML ( which is the only GPU choice it gives me ) and I am seeing that the GPU does NOT work. It uses the CPU no matter what choice I set it to. Also, It ramps up the CPU to 70% usage. Also, if I set the frame count to capture every 2 frames instead of just 1, then it lowers the CPU usage to 60%. But that is the only difference.

RAM : 64 Gigs
GPU: NVidia GTX 1650ti
CPU : i5-10300H
